src/styles/main.scss

```
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./utils/reset";
+@import "./utils/variables";
```

```suggestion
@import "./utils/variables";
@import "./utils/normalize";
```
Rename reset. scss to normalize.scss and
change the order of import files, because you need variables in it